examples/mysql/test_all

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -e
+
+export DATABASE_URL="mysql://travis@localhost/diesel_test"
```

Comment:
We shouldn't hard-code the `travis` user here. I'd leave the user off and see if CI complains. If it does, let's only set this if `TRAVIS_RUST_VERSION` is unset.